client/tsconfig.json

```
@@ -21,7 +21,8 @@
     "src/**/*",
     "utils/**/*",
     "tools/**/*",
-    "config/**/*"
+    "config/**/*",
+    "../shared/config/curriculum.ts"
```

```suggestion
    "config/**/*"
```
The client shouldn't compile the shared folder. It just consumes the .js and .d.ts files that `pnpm run create:shared` produces.